pegasus/sites.v3/code.org/public/ai.md.erb.partial

```diff
@@ -15,13 +15,18 @@ social:
   "og:video:height": ""
   "og:video:type": ""
 ---
+
+<%
+  show_tune_in_link = DCDO.get('show_2020_ethics_ai_panel_live_links', true)
```

Comment:
should this variable be used below?

Reply:
Oops sorry, was switching back and forth between having a variable and just getting it twice...will fix.